packages/spruce-next-helpers/src/skillskit/next/DevControls.js

```diff
@@ -42,16 +44,17 @@ class DevControls extends Component {
 
 		return (
 			<div {...props} className={`dev_controls ${className || ''}`}>
-				<select
+				<Select
 					className={`select`}
 					label="Jump to Role"
 					onChange={this.onChangeRole}
-				>
-					<option value="">{`Current: ${auth.role}`}</option>
-					<option value="owner">Owner</option>
-					<option value="teammate">Teammate</option>
-					<option value="guest">Guest</option>
-				</select>
+					options={{
```

Comment:
@jtini @brandonminch it's probably worth mentioning that this API (using an object to define options) probably needs some expansion to specify default, etc., in case you ever want to use it in an uncontrolled manner

Reply:
yo @gcpantazis - I was thinking you would set "value" or "defaultValue"  to handle selections.

Reply:
Discussed this with Tay on Slack but bringing it here just to close the loop:

Yeah! Wasn't really thinking about `defaultValue`. That's the main thing for uncontrolled, but options are super wonky, and sometimes you end up doing crazy stuff like:

`<option value="" selected disabled hidden>Choose here</option>`

... to get it to do exactly what you want. We don't need to handle that stuff right now, can cross that bridge when we come to it  😄